### PR TITLE
[manila-nanny] add nanny for checking affinity

### PR DESCRIPTION
**Pull request by chuan137**
The new nanny is merged here https://github.com/sapcc/openstack-nannies/pull/78

### CODE REVIEW

File: openstack/manila-nanny/templates/deployment.yaml [hunk 1]
```diff
@@ -282,4 +282,31 @@ spec:
...
+            - kubernetes-entrypoint
+          env:
+            - name: COMMAND
+              value: "pyreloader /scripts/manila-check-affinity.py"
+            - name: MANILA_NANNY_CONFIG
+              value: /manila-etc/manila.conf
+            - name: MANILA_NANNY_INTERVAL
+              value: {{ .Values.nannies.affinity.interval | default .Values.interval | quote }}
+            - name: MANILA_NANNY_PROMETHEUS_PORT
+              value: {{ .Values.nannies.snapshot_missing.prometheus_port | quote }}
```
Comment (Carthaca) [inline openstack/manila-nanny/templates/deployment.yaml:300]:
```suggestion
              value: {{ .Values.nannies.affinity.prometheus_port | quote }}
```

File: openstack/manila-nanny/templates/deployment.yaml [hunk 2]
```diff
@@ -282,4 +282,31 @@ spec:
...
+            - name: PYTHONUNBUFFERED
+              value: "1"
+            - {{ include "envNamespace" . | indent 14 | trim }}
+            - {{ include "envDependencyService" . | indent 14 | trim  }}
+            {{- include "envSentry" . | nindent 12 }}
+          volumeMounts:
+            - {{ include "mountManilaConfig" . | nindent 14 | trim }}
+          resources:
+            {{- .Values.nannies.snapshot.resources | default .Values.resources | toYaml | nindent 12 }}
```
Comment (Carthaca) [inline openstack/manila-nanny/templates/deployment.yaml:309]:
```suggestion
            {{- .Values.nannies.affinity.resources | default .Values.resources | toYaml | nindent 12 }}
```

File: openstack/manila-nanny/templates/deployment.yaml [hunk 3]
```diff
@@ -282,4 +282,31 @@ spec:
...
+          {{ include "promPort" .Values.nannies.snapshot | nindent 10 | trim }}
```
Comment (Carthaca) [inline openstack/manila-nanny/templates/deployment.yaml:310]:
```suggestion
          {{ include "promPort" .Values.nannies.affinity | nindent 10 | trim }}
```

File: openstack/manila-nanny/values.yaml [hunk 4]
```diff
@@ -88,6 +88,11 @@ nannies:
     interval: 1800
     prometheus_port: 9505
 
+  affinity:
+    enabled: true
+    interval: 86400
+    prometheus_port: 9505
```
Comment (Carthaca) [inline openstack/manila-nanny/values.yaml:94]:
```suggestion
    prometheus_port: 9506
```

**Review by Carthaca — CHANGES_REQUESTED** [review]
Good morning, Mister copy&paste ;)

**Comment by chuan137** [comment]
> Good morning, Mister copy&paste ;)

😓

**Comment by chuan137** [comment]
add aggregation rules and alerts
playbook is added here https://github.wdf.sap.corp/cc/documentation-operation/pull/1907/files

**Review by Carthaca — APPROVED** [review]

**Pull request merged**